src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -63,8 +63,16 @@
 
     private AgencyAndId stop;
 
+    /**
+     * The headsign of the alert
+     */
     private String direction;
```

Comment:
That is used in the AlertPatcher API endpoint (where JSON is deserialized into AlertPatches). As I don't know how and where that is used, I think we should keep the code backwards compatible.